src/shared/components/AssetImage/AssetImage.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactElement, useEffect, cloneElement, useState, useMemo } from 'react'
+
+import { AllChannelFieldsFragment, AssetAvailability, VideoFieldsFragment } from '@/api/queries'
+import { useStorageProviders } from '@/hooks'
+import { createStorageNodeUrl } from '@/utils/asset'
+
+type AssetImageProps = { entity?: AllChannelFieldsFragment | VideoFieldsFragment | null; component: ReactElement }
+
+const AssetImage = ({ entity, component }: AssetImageProps) => {
+  const [assetUrl, setAssetUrl] = useState<string | undefined>(undefined)
+  const { getStorageProvider } = useStorageProviders()
+  const assetData = useMemo(() => {
+    if (entity && entity.__typename === 'Channel') {
+      return {
+        title: entity.title,
```

Comment:
What do we need the title for?

Reply:
Sorry, previously I tried with `img` element and I used it for `alt`

Reply:
:+1: